app/editor/src/features/admin/work-orders/constants/columns.tsx

```diff
@@ -14,7 +14,20 @@ export const getColumns = (
     label: 'Content',
     name: 'configuration',
     width: 4,
-    cell: (cell) => <CellEllipsis>{JSON.stringify(cell.original.configuration)}</CellEllipsis>,
+    cell: (cell) => (
+      <CellEllipsis
+        className="link"
+        onClick={(e) => {
+          if (cell.original.configuration.contentId) {
+            e.preventDefault();
```

Comment:
Don't want it to load the work order form, we want it to go directly to the content.